README.md

```diff
@@ -50,57 +50,92 @@ Each image (apart from `ci-common`) is available for multiple VFX Platform Years
+as the major version number, and a minor version number to indicate minor
+changes in the Docker Image that still point to the same calendar year version,
+e.g. `2019.0` would be followed if necessary by a `2019.1` version.
+The minor version here does *not* point to a calendar month or quarter, it is
+solely to express that the image has changed internally. We could also have a
+patch version.
 
 ### Image Tags
 
-The most precise version tag is the `ASWF_VERSION` of the image, e.g. `aswf/ci-base:2019.0`, but it is recommended to use the `ASWF_VFXPLATFORM_VERSION` as the tag to use in CI pipelines, e.g. `aswf/ci-openexr:2019`.
-
-The `latest` tag is pointing to the current VFX Platorm year images, e.g. `aswf/ci-openexr:latest` points to `aswf/ci-openexr:2019.0` but will be updated to point to `aswf/ci-openexr:2020.0` in the calendar year 2020.
+The most precise version tag is the `ASWF_VERSION` of the image, e.g.
+`aswf/ci-base:2019.0`, but it is recommended to use the
+`ASWF_VFXPLATFORM_VERSION` as the tag to use in CI pipelines, e.g.
+`aswf/ci-openexr:2019`.
 
+The `latest` tag is pointing to the current VFX Platform year images, e.g.
+`aswf/ci-openexr:latest` points to `aswf/ci-openexr:2019.0` but will be
+updated to point to `aswf/ci-openexr:2020.0` in the calendar year 2020.
 
 ## Testing Images
 
-There is another dockerhub organisation with copies of the `aswf` docker images called `aswftesting`, images published there are for general testing and experimentations. Images can be pushed by any fork of the official repo as long as the branch is called `testing`. Images in this org will change without notice and could be broken in many unexpected ways!
+There is another Docker Hub organization with copies of the `aswf` Docker
+images called `aswftesting`, images published there are for general testing and
+experimentation. Images can be pushed by any fork of the official repo as long
+as the branch is called `testing`. Images in this org will change without
+notice and could be broken in many unexpected ways!
 
-To get write access to the `aswftesting` dockerhub organisation you can open a jira issue [there](https://jira.aswf.io).
+To get write access to the `aswftesting` Docker Hub organization you can open
+a Jira issue [there](https://jira.aswf.io).
 
 ### Status
-As of June 2020 there are full 2018, 2019 and 2020 [VFX Platform](https://vfxplatform.com) compliant images. And a preview of 2021
-with devtoolset-9.1 (GCC-9.3.1) and clang-10, N.B. that the built-in CUDA-10.2 is currently incompatible with GCC-9, hoping to update to
-CUDA-11 as soon as possible to restore CUDA support in 2021 images.
 
+As of June 2020 there are full 2018, 2019 and 2020
+[VFX Platform](https://vfxplatform.com) compliant images. And a preview of 2021
+with devtoolset-9.1 (GCC-9.3.1) and clang-10, N.B. that the built-in CUDA-10.2
+is currently incompatible with GCC-9, hoping to update to CUDA-11 as soon as
+possible to restore CUDA support in 2021 images.
```

Comment:
Isn't this done now with latest aswf-docker version?

Reply:
From the changelog:
> VFX 2021 images are now built against CUDA-11.1.
So I guess this remark can be removed.